scripts/src/test/bash/functions

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
maybe this file should have a different name (e.g. `test-functions`, ``)? Otherwise we might end up in confusions if someone talks about `functions` and it is unclear if he means the productive functions or the test functions. We also have a functions documentation and some places that just talk about `functions` what is currently a unique file and so it is clear what we are talking about.

Reply:
I created #868 for this to prevent blocking the merge.